aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/main/Dialog.java

```diff
@@ -169,4 +170,11 @@
      * @return One or more {@code Tab} annotations
      */
     Tab[] tabs() default {};
+
+    /**
+     * For the tabbed TouchUI dialog layout, enumerates the panels to be rendered
+     * @see AccordionPanel
+     * @return One or more {@code Accordion} annotations
```

Comment:
"Zero or more {@code AccordionPanel} annotatons"
"Zero" because this field is not required (as there may be tabs instead)
And please add same correction to the "tabs"above

Reply:
Fixed